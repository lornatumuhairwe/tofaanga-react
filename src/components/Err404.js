import React from 'react';

const Err404 = () => (
  <div>
    <h1>Err, route not found</h1>
    <p>Back to <a href="/">Home</a></p>
  </div>)

export default Err404;
