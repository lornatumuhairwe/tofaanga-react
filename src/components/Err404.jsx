import React from 'react';
const Err404 = () => (
  <div>
    <h1>Error, route not found</h1>
    <p>Back to <a href="/">Home</a></p>
  </div>);

export default Err404;
