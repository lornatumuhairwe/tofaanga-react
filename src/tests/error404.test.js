import React from 'react';
import renderer from 'react-test-renderer';
import Err404 from "../components/Err404";



describe('search snapshot', () => {
    it('matches snapshot', () => {
            const errHandler = renderer.create(<Err404/>);
            expect(errHandler.toJSON()).toMatchSnapshot();
        },

    );
});
