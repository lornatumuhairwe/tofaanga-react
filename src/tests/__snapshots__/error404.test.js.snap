// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search snapshot matches snapshot 1`] = `
<div>
  <h1>
    Error, route not found
  </h1>
  <p>
    Back to 
    <a
      href="/"
    >
      Home
    </a>
  </p>
</div>
`;
